internal/httpclient/model_self_service_saml_url.go

```diff
@@ -0,0 +1,138 @@
+ *
+ * API version: 1.0.0
+ * Contact: hi@ory.sh
+ */
+
+// Code generated by OpenAPI Generator (https://openapi-generator.tech); DO NOT EDIT.
+
+package client
+
+import (
+	"encoding/json"
+)
+
+// SelfServiceSamlUrl struct for SelfServiceSamlUrl
+type SelfServiceSamlUrl struct {
+	// SamlAcsURL is a post endpoint to handle SAML Response  format: uri
+	SamlAcsUrl string `json:"saml_acs_url"`
+	// SamlMetadataURL is a get endpoint to get the metadata  format: uri
+	SamlMetadataUrl string `json:"saml_metadata_url"`
+}
+
+// NewSelfServiceSamlUrl instantiates a new SelfServiceSamlUrl object
+// This constructor will assign default values to properties that have it defined,
+// and makes sure properties required by API are set, but the set of arguments
+// will change when the set of required properties is changed
+func NewSelfServiceSamlUrl(samlAcsUrl string, samlMetadataUrl string) *SelfServiceSamlUrl {
+	this := SelfServiceSamlUrl{}
+	this.SamlAcsUrl = samlAcsUrl
+	this.SamlMetadataUrl = samlMetadataUrl
+	return &this
+}
+
+// NewSelfServiceSamlUrlWithDefaults instantiates a new SelfServiceSamlUrl object
+// This constructor will only assign default values to properties that have it defined,
+// but it doesn't guarantee that properties required by API are set
+func NewSelfServiceSamlUrlWithDefaults() *SelfServiceSamlUrl {
+	this := SelfServiceSamlUrl{}
+	return &this
+}
+
+// GetSamlAcsUrl returns the SamlAcsUrl field value
+func (o *SelfServiceSamlUrl) GetSamlAcsUrl() string {
+	if o == nil {
+		var ret string
+		return ret
+	}
+
+	return o.SamlAcsUrl
+}
+
+// GetSamlAcsUrlOk returns a tuple with the SamlAcsUrl field value
+// and a boolean to check if the value has been set.
+func (o *SelfServiceSamlUrl) GetSamlAcsUrlOk() (*string, bool) {
+	if o == nil {
+		return nil, false
+	}
+	return &o.SamlAcsUrl, true
+}
+
+// SetSamlAcsUrl sets field value
+func (o *SelfServiceSamlUrl) SetSamlAcsUrl(v string) {
+	o.SamlAcsUrl = v
+}
+
+// GetSamlMetadataUrl returns the SamlMetadataUrl field value
+func (o *SelfServiceSamlUrl) GetSamlMetadataUrl() string {
+	if o == nil {
+		var ret string
+		return ret
+	}
+
+	return o.SamlMetadataUrl
+}
+
+// GetSamlMetadataUrlOk returns a tuple with the SamlMetadataUrl field value
+// and a boolean to check if the value has been set.
+func (o *SelfServiceSamlUrl) GetSamlMetadataUrlOk() (*string, bool) {
+	if o == nil {
+		return nil, false
+	}
+	return &o.SamlMetadataUrl, true
+}
+
+// SetSamlMetadataUrl sets field value
+func (o *SelfServiceSamlUrl) SetSamlMetadataUrl(v string) {
+	o.SamlMetadataUrl = v
+}
+
+func (o SelfServiceSamlUrl) MarshalJSON() ([]byte, error) {
+	toSerialize := map[string]interface{}{}
+	if true {
```

Comment:
This condition will always be true, so it can be removed.

Same for the other ones below.